addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -121,8 +126,15 @@ export async function loadEmoji() {
 export const EMOJI_PER_ROW = 9;
 
 export class EmojiPicker extends Component {
-    static props = ["onSelect", "close", "onClose?", "storeScroll?"];
-    static defaultProps = { onClose: () => {} };
+    static props = [
+        "onSelect",
+        "close",
+        "onClose?",
+        "restorePositionOnSelect?",
+        "storeScroll?",
+        "className?",
+    ];
```

Comment:
I'd expect the list of props to not change in PR. At most just a single prop to manage visibility, but even that I'm not entirely sure it's needed. `storeScroll` should be internal code of `EmojiPicker`.